src/main/resources/db/migration/data/dev/R__1_1__locations.sql

```diff
@@ -1,2 +1,8 @@
-insert into locations (location_id, added_at, location_type, nomis_agency_id, site_name) select * from (select uuid('709fbee3-7fe6-4584-a8dc-f12481165bfa'), current_timestamp, 'PR', 'GNI', 'HMP Grendon') as tmp where not exists (select location_id from locations where location_id = '709fbee3-7fe6-4584-a8dc-f12481165bfa');
-insert into locations (location_id, added_at, location_type, nomis_agency_id, site_name) select * from (select uuid('13c46837-c5c9-45a4-83d5-5a0d1438ff3c'), current_timestamp, 'PR', 'WYI', 'HMYOI Wetherby') as tmp where not exists (select location_id from locations where location_id = '13c46837-c5c9-45a4-83d5-5a0d1438ff3c');
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
This line just makes sure the relevant UUID functions are available (if not already).  It should work (I think).